protos/organization_transaction.proto

```diff
@@ -0,0 +1,83 @@
+// Copyright 2018 Bitwise IO
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+syntax = "proto3";
+
+import "organization.proto";
+
+message CreateOrganization {
+	
+    // The name of the organizaiton
+    string name = 1;
+
+    // Type of the organization
+    Organization.OrganizationType organization_type = 2;
+
+    // The industry of the organization
+    string industry = 3;
+
+    // Ticker symbol for organizations with organization type 'TRADING FIRM'
+    // uniquely identifying the organization;
+    // Pricing source for organizations with organization type 'PRICING SOURCE'
+    // is a four-letter code representing organizations responsible for
+    // providing quotes to the market.
+    // Ticker symbol or pricing source provided dependent on organization type
+    string organization_id = 4;
+
+    // List of participant public keys and their roles
+    // within the organization
+    repeated Organization.Authorization authorizations = 5;
+}
+
+message UpdateOrganization {
+
+    // The name of the organization
+    string name = 1;
+
+    // Ticker symbol for organizations with organization type 'TRADING FIRM'
+    // uniquely identifying the organization;
+    // Pricing source for organizations with organization type 'PRICING SOURCE'
+    // is a four-letter code representing organizations responsible for
+    // providing quotes to the market.
+    // Ticker symbol or pricing source provided dependent on organization type
+    string organization_id = 2;
+
+    // Type of the organization
+    Organization.OrganizationType organization_type = 3;
```

Comment:
If we are using the ticker or pricing source as id, depending on the organization type. We should not allow organization type to be updated, should we?

Reply:
I was thinking the same thing when I went through and updated this portion... Maybe we should discuss implementing the update organization transaction seeing as how a lot of the information to how we identify the org is alterable based on this transaction.